sphinx_needs/needsfile.py

```diff
@@ -131,6 +131,21 @@ def load_json(self, file) -> None:
 
             self.log.debug(f"needs.json file loaded: {file}")
 
+    def write_jsons(self, needs_file, needs_path) -> None:
```

Comment:
If you have the loop for the needs outside the `write_jsons ` function, why can't you just use the existing `write-json`?